operations/rack_resiliency/manage_critical_services.md

```diff
@@ -0,0 +1,404 @@
+# Editing/Deleting critical services from static ConfigMap (`rrs-mon-static`)
+
+## Overview
```

Comment:
This page should also address adding a critical service. I realize that this is done with the API/CLI, but if someone is looking at this page, it is weird to see all of the procedures for list/delete/edit and have no mention of add. There should be some blurb about how adding is done using the API/CLI, with a link to the appropriate place in the docs that has details on how to do that.